src/main/java/net/coderbot/iris/uniforms/CommonUniforms.java

```diff
@@ -31,14 +26,17 @@
 import net.minecraft.util.math.Vec3d;
 import net.minecraft.world.LightType;
 
+import static net.coderbot.iris.gl.uniform.UniformUpdateFrequency.PER_FRAME;
+import static net.coderbot.iris.gl.uniform.UniformUpdateFrequency.PER_TICK;
+
 public final class CommonUniforms {
 	private static final MinecraftClient client = MinecraftClient.getInstance();
 
 	private CommonUniforms() {
 		// no construction allowed
 	}
 
-	public static void addCommonUniforms(UniformHolder uniforms, IdMap idMap) {
+	public static void addCommonUniforms(LocationalUniformHolder uniforms, IdMap idMap) {
```

Comment:
Does this need to be locational?

Reply:
yep, see https://github.com/IrisShaders/Iris/pull/238#discussion_r611900460